app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx

```diff
@@ -106,6 +111,19 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ expanded, setEx
     }
   };
 
+  const determineActiveToggles = (): string => {
```

Comment:
Is this the display text values? Looks like something could be a simple dictionary in a constants folder.